### PR TITLE
Feature / Delete target feature

### PR DESCRIPTION
**Pull request by adlamas**
### Description
As a user I should be able to delete a previously created target so it's no longer available in the map

### Trello card
[Link](https://trello.com/c/IrtMrFB2/10-2-as-a-user-i-should-be-able-to-delete-a-previously-created-target-so-its-no-longer-available-in-the-map)

### Figma / Image
[Link](https://trello.com/1/cards/63b86eb0ee1c08004a79a5c5/attachments/63b86eb1ee1c08004a79a7e0/previews/63b86eb1ee1c08004a79a7e6/download/Screen_Shot_2016-08-01_at_4.25.39_PM.png)

### CODE REVIEW

File: src/services/deleteTarget/deleteTargetService.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+import Endpoints from 'constants/endpoints';
+
+const deleteTargetService = async (id: number) => {
+  const deleteTargetEndpoint: string = `${process.env.REACT_APP_API_BASE_URL}${Endpoints.DELETE_TARGET}/${id}`;
+
+  try{
```
Comment (joaquincollado) [inline src/services/deleteTarget/deleteTargetService.tsx:7]:
```suggestion
  try {
```

File: src/services/deleteTarget/deleteTargetService.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        headers: {
+          'Content-Type': 'application/json',
+          'access-token': sessionStorage.getItem('api_key_access-token'),
+          'client': sessionStorage.getItem('api_key_client'),
+          'uid': sessionStorage.getItem('api_key_uid')
+        }
+      }
+    )
+    return res;
+  }catch(e: any){
```
Comment (joaquincollado) [inline src/services/deleteTarget/deleteTargetService.tsx:20]:
```suggestion
  } catch(e: any) {
```

**Review by joaquincollado — APPROVED** [review]
LGTM, are you using any code formater like prettierrc?

**Comment by adlamas** [comment]
@joaquincollado  No, but I can add it eventually 😬

**Pull request merged**